src/lib/workspace/Subsetting/SubsetDownloadModal.scss

```diff
@@ -0,0 +1,43 @@
+.SubsetDownloadModal {
+  .css-ro2cao-Modal {
+    overflow: hidden;
+  }
+
+  .css-ob34sg-Modal {
```

Comment:
Ok, let's put in on the docket for this Friday then!

More vertical space there might be nice! I tried to compact it as much as I thought reasonable to make room for the main content (var picker and table), since now that the full modal doesn't scroll, the content can only take up so much of the modal height.

Good question! Those class names are autogenerated from the libraries that render those components (and maybe ultimately by emotion's css prop? Not quite sure). They seem to be consistent, and let's hope they stay that way!

Reply:
I am always a little uncertain about using generated class names as well. `styleOverrides` does not allow you to change all of the CSS I needed to for this layout unfortunatley. `styleOverrides` currently only gives you access to a specific subset of CSS rules to modify. If we want to extend `styleOverrides` every time we need to change something it won't yet let us, it could end up encompassing most CSS rules anyway, which seems to be against the idea. In other words, I think this is a deeper issue.

Instead I've added `className`s to the appropriate components and used those instead of their generated classes. Does that work for you, @dmfalke?

Reply:
We started a similar discussion in [this comment](https://github.com/VEuPathDB/CoreUI/pull/91#issuecomment-1182420660). Would it be worth us discussing it in a meeting? Either with just whomever is interested or at one of the dev meetings? It might be helpful to talk through generally when classes would be best vs putting a bunch of css into the component

Reply:
@chowington I understand your point, but I'm also not sure about having both `styleOverrides` and a class-based mechanism for customizing the style. To your point on always having to add to `styleOverrides` anytime we want to change something... the same can be said about class names.

While it's true that using traditional CSS is much more powerful, it also exposes a new contract based on the internal DOM structure of the component, which makes internal changes to the component more difficult to do in a way that doesn't break consumers of the component. The nice thing about `styleOverrides` is that it's an intentional contract that is backed by TypeScript interfaces, so it's easier to see when things break.

I agree we should discuss this in a meeting. There are pros and cons to consider.

Reply:
Thanks @dmfalke, I see your point about classNames becoming too dependent on the internal DOM structure. Definitely worth discussing!